### PR TITLE
Updated README.md and footer.html

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by fuyuton** [comment]
I think I will leave twit in the sense that I talk to the developer, but how is it?

File: public/tpl/footer.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
                 <address class="margin-bottom-40"> 
                     <br>Open an Issue: <a href="https://github.com/nekonium/explorer/issues">github</a>
                     <br> <a href="mailto:nekonium@nekonium.org">Email bug reports</a>
-                    <br> <a href="https://twitter.com/NekoniumDev">Twit</a>
+                    <br> <a href="https://twitter.com/NekoniumDev">Twitter</a>
```
Comment (fuyuton) [inline public/tpl/footer.html:26]: I think I will leave twit in the sense that I talk to the developer, but how is it?

Comment (ghost) [reply public/tpl/footer.html:26]: @fuyuton Twitter seems like a more reasonable description for the link, but i guess that it does not matter that much.

Comment (fuyuton) [reply public/tpl/footer.html:26]: That may be better. Thanks Pull request.

**Pull request merged**